src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -52,7 +52,7 @@
          validation errors on workload dependencies), we are instead manually
          referencing what we need out of the macos SDK. -->
     <PackageDownload Include="Microsoft.macOS.Ref" Version="[$(MicrosoftmacOSRefVersion)]" />
-    <Reference Include="$(NuGetPackageRoot)microsoft.macos.ref\$(MicrosoftmacOSRefVersion)\ref\net6.0\Xamarin.Mac.dll" />
+    <Reference Include="$(NuGetPackageRoot)microsoft.macos.ref\$(MicrosoftmacOSRefVersion)\ref\net6.0\Microsoft.macOS.dll" />
```

Comment:
@sandyarmstrong this is the same change as you're making in https://github.com/dotnet/roslyn/pull/61250

Is this a problem? Or do we need to wait for the 6.0.300 bump in VS Mac?

Reply:
Ping @sandyarmstrong, could you please take a look at this and let me know if it's OK? Thanks

Reply:
You should merge https://github.com/dotnet/roslyn/pull/61250 first. But then we will not be able to insert Roslyn with this change for at least a few days while we finish up the work to bump VSMac to use the 6.0.300 SDK and the newer macos workload.

Reply:
I merged https://github.com/dotnet/roslyn/pull/61250, we should be ready to consume it in VSMac in the next few days.